tests/end_to_end/test_docker_monitoring/test_docker_monitoring.py

```diff
@@ -0,0 +1,61 @@
+import json
+import os
+import re
+from tempfile import gettempdir
+import pytest
```

Comment:
```suggestion
import json
import os
import re
import pytest
from tempfile import gettempdir
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2948/commits/de272050ca3415a878861a8ee0461883c3936f17